### ISSUE TITLE
Problème d'aiguillage

### ISSUE THREAD

**Issue by Rouergue** [EditionsDuRouergue/lerouergue#433]
Les deux brunes en poche de mars apparaissent dans la catégorie polars sur le site.
Pourtant l’affectation comptable dans Oléa est bonne :

![Unknown](https://user-images.githubusercontent.com/10072454/116526260-b3a16300-a8d9-11eb-88b9-591c5e9a10c1.png)

![Capture d’écran 2021-04-29 à 10 56 02](https://user-images.githubusercontent.com/10072454/116526267-b69c5380-a8d9-11eb-992c-3d58b9a2964e.png)

**Comment by Fchallier** [comment]
Je remarque le dans Mistral il a toujours la série  "Assassins sans visages" c'est pour cela qu'il remonte dans polar. 

Il probablement un un mauvais mapping entre Olea et Mistral, je vais voir cela avec Charles

**Comment by Rouergue** [comment]
Oui, c'est un problème que nous avons maintes fois soulevé et pour lequel nous n'avons jamais eu de réponse... 
Ci-dessous les échanges de mail à ce sujet et les demandes faites à Pauline.
_______________________
mail du 29/03/21

Bonjour Pauline,
 
Nous avons un souci récurrent d’aiguillage au niveau d’Olea pour deux de nos livres littérature poche qui se retrouvent dans la collection polar assassins sans visages. Nathalie a changé manuellement plusieurs fois l’info sur Olea mais, visiblement, le sous-département indique toujours la mauvaise collection. Comment faire pour que ce soit modifié durablement et que cela se répercute automatiquement dans Mistral (dont dépend la synchronisation avec notre site), stp ? Merci beaucoup pour ton aide.
Bonne journée,
 
Mathilde Gautron-Davienne
ROUERGUE – 04 88 65 90 03
 
 
![Unknown-1](https://user-images.githubusercontent.com/10072454/116557980-4274a680-a8ff-11eb-98ac-1fa5591448f0.png)

**Comment by Rouergue** [comment]
As-tu eu des nouvelles de Charles, par rapport à ce problème ? Car les 2 poches (litté) sont toujours et encore dans la coll. polar "Assassins sans visages".

![Capture d’écran 2021-05-19 à 09 22 18](https://user-images.githubusercontent.com/10072454/118772417-02f90480-b884-11eb-9a1b-913b7d52f5db.png)

**Comment by Rouergue** [comment]
Toujours pas de changement sur le site... 
Tu aurais un moment dans la journée pour qu'on puisse faire un point rapide sur les problèmes qui restent en suspens ? J'aurais également quelques petites questions par rapport à youtube...
Merci !

**Comment by Rouergue** [comment]
Je reviens sur ce problème qui ressemble à une série à l'arlésienne dont on ne voit pas le bout !

J'ai vérifié sur Olea et Mistral, la mention "ASSASSINS SANS VISAGES" a bien été supprimé du sous-département 2 (pour Olea) et de la série (pour Mistral). Tout devrait être bon donc, normalement... Or ces deux poches se retrouvent encore et toujours dans la coll. Assassins sans visages sur le site. 

![Capture d’écran 2021-09-29 à 12 17 18](https://user-images.githubusercontent.com/10072454/135250572-fe800616-e998-4d8a-aa68-98e8ee11f3a2.png)

![Capture d’écran 2021-09-29 à 12 17 46](https://user-images.githubusercontent.com/10072454/135250583-9315c401-9752-495a-be9a-3bea8aeefd8f.png)

**Comment by Fchallier** [comment]
à jour depuis la correction dans mistral non  ?
